src/bundle/Controller/FormController.php

```diff
@@ -96,7 +105,7 @@ protected function getTemplateVars(Request $request, TwoFactorTokenInterface $to
     {
         $config = $this->twoFactorFirewallContext->getFirewallConfig($token->getProviderKey());
         $pendingTwoFactorProviders = $token->getTwoFactorProviders();
-        $displayTrustedOption = $this->trustedFeatureEnabled && (!$config->isMultiFactor() || 1 === count($pendingTwoFactorProviders));
+        $displayTrustedOption = $this->trustedFeatureEnabled && (!$config->isMultiFactor() || 1 === \count($pendingTwoFactorProviders));
```

Comment:
I have to say, until today I've never understoods what's the thing with with prefixing built-in global functions with `\`. Could you explain to me?

Btw. further down there are some function calls without a `\` prefix, which confuses me even more when `\` is added and when it isn't. Cannot make out a pattern that makes sense to me.

Reply:
1. When a global function is called from inside in namespace, it has a tiny little overhead (only for the first call if OPCache is enabled, and even then not really measurable) because PHP has to check if there's a function with the same name inside the current namespace before calling the global one.

2. There are some "special" core functions with compiler optimized code that only gets triggered when explicitly called as global function. The difference for these *is* measurable, so often only these get prefixed. But it's hard to remember which they are, so it's best to let a tool like PHP-CS-Fixer or [Php Inspections ​(EA Extended)](https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended-) (or even [Ultimate](https://plugins.jetbrains.com/plugin/10215-php-inspections-ea-ultimate-)) do this for you.<br>
A list of these functions can be found here, for example (contains links to the PHP sources, too): https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/9b3716c623bde9f497022675acef31a60fa19561/src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php#L357-L397

Prefixing the functions with `\` or using them via `use function ...` makes no difference, it's just a matter of taste.

Reply:
As @jdreesen said, that's a micro optimization when calling this function from a namespace.
I will revert it to avoid confusion

Reply:
Ok, I just wanted to understand. So let's keep it, but then we should make sure we have this consistent within the codebase.